.eslintignore

```diff
@@ -0,0 +1,3 @@
+build
```

Comment:
Is there a 'build' folder?

Reply:
Unless this is like a default setting then I guess its fine to leave it in, although the 'builds' are in packages/*/dist

Reply:
Yea this is a default setting, I'll remove this